lms/services/digest.py

```diff
@@ -75,9 +75,9 @@ def send_instructor_email_digests(  # pylint:disable=too-many-arguments
             else:
                 task_done_key = None
 
-            send_template.delay(
+            send.delay(
                 task_done_key=task_done_key,
-                template_name="instructor-email-digest",
+                template="lms:templates/email/instructor_email_digest/",
```

Comment:
The template for an email is actually a folder containing `subject.jinja2` and `body.html.jinja2` files. In future it might also contain `body.txt.jinja2` if we also want to do our own plain text version of the email (for now I'm relying on Mailchimp's ability to turn HTML into plain text for us).